File: Physio/Physio.Core/Exercise.cs
namespace Physio.Core;

public class Exercise
{
    public Guid Id { get; set; }
    public required string Name { get; set; }
    public string? Description { get; set; }
    public int EstimatedDuration { get; set; }
    public int Repetitions { get; set; }
    public DifficultyLevel Difficulty { get; set; }
    public string? Instructions { get; set; }
}

public enum DifficultyLevel
{
    Easy = 0,
    Medium = 1,
    Hard = 2
}

File: Physio.Android/app/src/main/java/com/core/physio/data/model/Exercise.kt
package com.core.physio.data.model

data class Exercise(
    val exerciseId: Int,
    val exerciseName: String,
    val description: String,
    val estimatedDurationMinutes: Int,
    val difficultyLevel: Int,
    val instructions: String,
    val repetitions: Int
) 